File: Disciplina-AlgProg/Exercicios/Exercicios/Lista 1 - Operadores Aritméticos/Atividade_B/Lst01qst12.py
import sys
sys.path.append(".")
from utils import getNumber, showPresentation, showResult

def main():
    showPresentation("Leia o salário de um trabalhador e escreva seu novo salário com um aumento de 25%.")

    #Entrada
    wage = getNumber("Digite o salário do colaborador: ")

    #Processamento
    newWage = wage * 1.25

    #Saída
    showResult(f"O novo salário é de {newWage}")

main()

File: Disciplina-AlgProg/Exercicios/Exercicios/Lista 1 - Operadores Aritméticos/Atividade_B/Lst01qst12.c
#include <stdio.h>

// Aumento de 25% no salário

int GetNumber(char message[]){
    int number;

    printf("%s", message);
    scanf("%d", &number);

    return number;
}

int main(){
    //Entrada
    int salary = GetNumber("Digite o salario do trabalhador: ");

    //Processamento
    float newSalary = salary * 1.25;

    //Saída
    printf("O novo salario do trabalhador e %.2fR$!", newSalary);

    return 0;
}